sda-download/api/s3/s3.go

```diff
@@ -188,6 +188,26 @@ func ListObjects(c *gin.Context) {
 	})
 }
 
+func getFileInfo(c *gin.Context) (fileInfo *database.FileInfo, err error) {
+	// Get file info for the given file path (or abort)
+	filename := c.Param("filename")
+	if !strings.HasSuffix(c.Param("filename"), ".c4gh") {
+		filename = c.Param("filename") + ".c4gh"
```

Comment:
Why append the suffix ".c4gh" to the filename? How if it's an encrypted file?

Reply:
The file extension was previously added by default (see [the remove line here](https://github.com/neicnordic/sensitive-data-archive/commit/32478c8f5e8fd77c4997b7ac7b21b5ef676de5f2#diff-481dfe13739d77ba443566773ab0564d718e309473b081baebbbc4acd355a7a8R193)), we now check that we don't get an extra suffix, thus allowing the client to require `file.c4gh` .